src/com/twitter/intellij/pants/compiler/PantsBuildProcessParametersProvider.java

```diff
@@ -27,6 +27,6 @@ public PantsBuildProcessParametersProvider(Project project) {
     if (PantsUtil.isPantsProject(myProject)) {
       throw new RuntimeException(PantsConstants.EXTERNAL_BUILDER_ERROR);
     }
-    return Lists.newArrayList();
```

Comment:
Ah this was meant to use the guava `Lists`

Reply:
I figured, but rather than add guava, I just used the base java methods.  We could also add guava (big fan!).

Reply:
guava is already used by intellij core, so there's no additional cost to use it.

Reply:
let's use guava then.